provision/aws/rds/aurora_create_peering_connection.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/aurora_common.sh
+
+sh ${SCRIPT_DIR}/../rosa_oc_login.sh
+
+export AWS_REGION=$(rosa describe cluster -c ${CLUSTER_NAME} -o json | jq -r .region.id)
+
+AURORA_VPC=$(aws ec2 describe-vpcs \
+  --filters "Name=cidr,Values=${AURORA_VPC_CIDR}" \
```

Comment:
This will look up all VPCs in the region matching the `${AURORA_VPC_CIDR}`. This will be problematic if 2 or more users provision Aurora in the same region with the same CIDR, eg. the default one.

Could we tag the VPCs with `${AURORA_CLUSTER}` name instead, and then look up via the tag? This should be unique within the region.

Same would apply for `aurora_delete_peering_connection.sh` and `aurora_delete.sh` scripts.

Reply:
> Could we tag the VPCs with `${AURORA_CLUSTER}` name instead, and then look up via the tag? This should be unique within the region.

Good point,  I'll add a tag.